### PR TITLE
fix(api-client): propagate schema to request body form-data table

### PR DESCRIPTION
**Pull request by DemonHa**
FIxes: #6816 

Form fields defined with enum values in the request body were not rendering as dropdown inputs in the form-data table. This happened because the schema information was not being properly propagated to the table component, preventing it from detecting enum constraints.

This PR ensures the request body schema is correctly passed down to the form-data table so that enum fields are recognized and rendered as dropdowns.

image

## Checklist

- [x] I explained why the change is needed.
- [x] I added a changeset. 
- [x] I added tests.
- [ ] I updated the documentation.

---

> [!NOTE]
> **Low Risk**
> Low risk UI/data-mapping change confined to request body form table rendering; main risk is mismatched row serialization (string/File) for edge-case examples, covered by added tests.
> 
> **Overview**
> Fixes form-data and URL-encoded request body tables so fields can render with schema-driven UI (e.g. enum dropdowns) by **passing the resolved request-body schema down** from `RequestBody.vue` into `RequestBodyForm.vue`.
> 
> Updates `getFormBodyRows` to accept an optional schema and **enrich each row** with per-property `schema`, `description`, and `isRequired` (with `$ref` resolution), and adds/extends unit tests to cover schema enrichment behavior.
> 
> <sup>Written by [Cursor Bugbot](https://cursor.com/dashboard?tab=bugbot) for commit 6de4811c1096aca00c7e8ab8f9175a2a4279206c. This will update automatically on new commits. Configure [here](https://cursor.com/dashboard?tab=bugbot).</sup>

### CODE REVIEW

File: packages/api-client/src/v2/blocks/request-block/helpers/get-form-body-rows.ts [hunk 1]
```diff
@@ -11,22 +20,47 @@ export const getFormBodyRows = (example: ExampleObject | undefined | null, conte
...
+  }): TableRow => {
+    const row: TableRow = { name, value, isDisabled }
+    if (!schemaWithProperties || !name) {
+      return row
+    }
+    const propSchema = resolve.schema(schemaWithProperties.properties?.[name])
+    row.schema = propSchema
+    row.description = propSchema?.description
+    row.isRequired = requiredSet?.has(name)
+    row.isDisabled = isDisabled
```
[the remark this answers is not in the document]
Comment (DemonHa) [reply packages/api-client/src/v2/blocks/request-block/helpers/get-form-body-rows.ts:44]: that is fineeee
